web/templates/stream_list_item.hbs

```diff
@@ -8,12 +8,16 @@
         </span>
         <div class="remove-button-wrapper">
             {{#if show_action_button}}
-                {{#if show_private_stream_unsub_tooltip}}
-                    {{> components/icon_button  icon="close" custom_classes="hidden-remove-button remove-button tippy-zulip-tooltip" intent="danger" aria-label=(t "Unsubscribe") data-tippy-content=(t "Use channel settings to unsubscribe from private channels.") }}
-                {{else if show_last_user_in_private_stream_unsub_tooltip}}
-                    {{> components/icon_button  icon="close" custom_classes="hidden-remove-button remove-button tippy-zulip-tooltip" intent="danger" aria-label=(t "Unsubscribe") data-tippy-content=(t "Use channel settings to unsubscribe the last user from a private channel.") }}
-                {{else if show_unsubscribe_tooltip}}
-                    {{> components/icon_button  icon="close" custom_classes="hidden-remove-button remove-button tippy-zulip-delayed-tooltip" intent="danger" aria-label=(t "Unsubscribe") data-tippy-content=(t "Unsubscribe") }}
+                {{#if show_unsubscribe_tooltip}}
```

Comment:
I was thinking we would instead use a better name like `show_unsubscribe_button` as tooltip feels weird as a different button, which does a different thing on clicking, is used.

Also, there is no need of having the last `else if show_unsubscribe_tooltip` block in the previous commit as well, you can just use `else` there.